### PR TITLE
CI: Azure timeout debug

### PR DESCRIPTION
**Pull request by mroeschke**
Parallel troubleshooting with https://github.com/pandas-dev/pandas/pull/45507#issuecomment-1018536106

### CODE REVIEW

File: ci/azure/windows.yml [hunk 1]
```diff
@@ -47,7 +48,7 @@ jobs:
         ENV_FILE: ci/deps/azure-windows-310.yaml
         CONDA_PY: "310"
         PATTERN: "not slow and not high_memory"
-        PYTEST_WORKERS: 2  # GH-42236
+        PYTEST_WORKERS: 1  # GH-42236
         PYTEST_TARGET: "pandas/tests/[i-z]*"
```
Comment (jbrockmendel) [inline ci/azure/windows.yml:52]: could also do a `-v` for these?

**Comment by mroeschke** [comment]
Possibly a candidate for the hanging behavior: https://github.com/pandas-dev/pandas/pull/45389

**Comment by mroeschke** [comment]
Yeah from the logs maybe the teardown step is hanging?

```
2022-01-22T17:25:12.8017367Z [gw0] PASSED pandas/tests/io/parser/test_unsupported.py::test_close_file_handle_on_invalide_usecols[python] 
2022-01-22T18:34:32.0699589Z ##[error]The operation was canceled.
```

**Comment by jbrockmendel** [comment]
> Yeah from the logs maybe the teardown step is hanging?

yah in the other PR i short-circuited the pyarrow case and it got past this OK, but now looks like its hanging on a totally unrelated offsets test.

**Comment by jreback** [comment]
@mroeschke still needed?

**Pull request closed without merging**